### PR TITLE
Wisconsin dates

### PR DESCRIPTION
**Pull request by codeLovingYogi**
Nearly 21 million gallons of ice cream are consumed by Wisconsinites each year.
Wisconsin is a leading producer of Ginseng in the United States.
Green Bay is known as the "Toilet Paper Capital" of the world.

### CODE REVIEW

File: states/wisconsin/elections.toml [hunk 1]
```diff
@@ -8,7 +8,50 @@ date = 2020-08-11
 name = "Congressional Primary"
 original_date = 2020-08-11
 
+[20200811.registration]
+# by 8/7 5pm at municipal clerk
+in_person_by = 2020-08-11T20:00:00
+postmarked_by = 2020-07-22
```
Comment (tannewt) [inline states/wisconsin/elections.toml:14]:
```suggestion
postmarked_by = 2020-07-22
online_by = 2020-07-22
```

File: states/wisconsin/elections.toml [hunk 2]
```diff
@@ -8,7 +8,50 @@ date = 2020-08-11
...
+sources = ["https://myvote.wi.gov/en-us/VoterDeadlines", "https://www.vote411.org/wisconsin"]
+
+[20200811.poll]
+in_person_by = 2020-08-11
```
Comment (tannewt) [inline states/wisconsin/elections.toml:20]:
```suggestion
in_person_start = 2020-08-11
in_person_by = 2020-08-11
```

File: states/wisconsin/elections.toml [hunk 3]
```diff
@@ -8,7 +8,50 @@ date = 2020-08-11
...
+
+# early voting considered in-person absentee and varies by municipal
+
+[20200811.absentee]
+# WI recommends mail in one week before
```
Comment (tannewt) [inline states/wisconsin/elections.toml:25]: We do a week's heads up by default with our reminders

File: states/wisconsin/elections.toml [hunk 4]
```diff
@@ -8,7 +8,50 @@ date = 2020-08-11
...
+
+
 [20201103]
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+in_person_by = 2020-11-03T20:00:00
+postmarked_by = 2020-10-14
```
Comment (tannewt) [inline states/wisconsin/elections.toml:44]:
```suggestion
postmarked_by = 2020-10-14
online_by = 2020-10-14
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Just a couple things. Thanks!

**Comment by codeLovingYogi** [comment]
Requested changes done!

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**